code/modules/client/client_procs.dm

```diff
@@ -223,6 +223,13 @@
 	if(holder)
 		src.control_freak = 0 //Devs need 0 for profiler access
 
+	// Update our windows and chat settings.
+	if(get_preference_value(/datum/client_preference/chat_position) == GLOB.PREF_YES)
+		update_chat_position(TRUE)
+
+	if(get_preference_value(/datum/client_preference/fullscreen_mode) != GLOB.PREF_OFF)
+		toggle_fullscreen(get_preference_value(/datum/client_preference/fullscreen_mode))
```

Comment:
It's unsafe to do it this early, as clients may be loaded before prefs are set up. The result will be that these will runtime and the toggles won't fire.